config/pmd/ruleset.xml

```diff
@@ -105,4 +105,10 @@
       <property name="cycloOptions" value="" />
     </properties>
   </rule>
+<!--  <rule ref="category/java/design.xml/ExcessiveMethodLength">-->
```

Comment:
Why remove this?

Reply:
It was added to resolve the exception but was not the part of the rules . i added it to test something and now removed it.